shaders/src/depth_main.fs

```diff
@@ -38,8 +40,20 @@ void main() {
 
 #if defined(HAS_VSM)
     // interpolated depth is stored in vertex_worldPosition.w (see main.vs)
-    highp float depth = exp(vertex_worldPosition.w);
+    highp float depth = vertex_worldPosition.w;
+    depth = exp(depth);
+    fragColor.xy = computeDepthMomentsVSM(depth);
+    fragColor.zw = vec2(0.0);
+    //fragColor.zw = computeDepthMomentsVSM(-1.0/depth);
```

Comment:
remove?

Reply:
no I want to keep it so it's easy to activate in `matdbg` -- e.g. when seeing artifacts with EVSM, it's nice to be able to quickly see if it's because of that.

Reply:
Then leave a comment because otherwise I'm sure I'll remove this random piece of code next time I see it since I will have forgotten why you want it.